config/locales/ja.yml

```
@@ -9,6 +9,15 @@ ja:
         format: "%u%n"
         delimiter: ","
         precision: 0
+  enumerize:
+    purchase:
+      delivery_time:
+        eight_to_twelve: 8時〜12時
+        twelve_to_two: 12時〜14時
```

```suggestion
        twelve_to_two: 12時〜14時
```
twoと14ってのがなんだかなー、という感じ